cmd/geth/chaincmd.go

```diff
@@ -341,6 +341,10 @@ func exportChain(ctx *cli.Context) error {
 		if first < 0 || last < 0 {
 			utils.Fatalf("Export error: block number must be greater than 0\n")
 		}
+		head := chain.CurrentBlock()
+		if uint64(last) > head.NumberU64() {
```

Comment:
More go-idiomatic form:
```suggestion
		if head := chain.CurrentBlock(); uint64(last) > head.NumberU64() {
```